helpers/configHelpers.py

```diff
@@ -28,5 +43,55 @@ def loadEnvFile(runType, fileString):
 
     if envDict is None:
         return {}
-    
+
     return envDict
+
+
+def loadEnvVars(runType):
+    """Loads a full set of configuration details from the both the default
+    configuration file and any environment specific settings.
+
+    Arguments:
+        runType {string} -- The current environment.
+
+    Returns:
+        ChainMap -- A ChainMap object combining the dictionaries returned from
+        the default config.yaml file (required) and an environment-specific
+        file (optional). The environment-specific details will override any
+        settings in the default file.
+    """
+    # Load env variables from relevant .yaml file
+    envDict = loadEnvFile(runType, 'config/{}.yaml')
+
+    # Overwrite/add any vars in the core config.yaml file
+    configDict = loadEnvFile(runType, None)
+
+    combinedConfig = ChainMap(envDict, configDict)
```

Comment:
If all you're doing is combining two dicts, this is faster (in versions 3.5+) as long as it's okay for values in the second dict to overwrite matching keys in the first.:

    {**envDict, **configDict}

I think ChainMap does the opposite when you flatten one to a dict. That, is it keeps values from the first. So the equivalent of:

    dict(ChainMap(envDict, configDict))

may be:

    {**configDict, **envDict}

Reply:
Thanks for the tip, I wasn't aware of that construction. And yes, the idea is explicitly to have the second dict overwrite values in the first (so you can create env-specific variables for production that overwrite the general settings used in local/production/qa branches, etc.)